test/app.spec.js

```diff
@@ -24,6 +24,11 @@ describe("App", () => {
     app({ model: {}, view: () => (html`<div>Hi</div>`) })
   })
 
+  // FIXME: if fails... this is not true
+  it.skip("allow all params to be optional", () => {
+    app()
```

Comment:
Because that's what the README says... We have to 1) change the README and specify what params are not optional or 2) change the code...